### PR TITLE
[v7r3] Hackathon fixes

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

File: docs/source/UserGuide/GettingStarted/InstallingClient/index.rst [hunk 1]
```diff
@@ -8,29 +8,39 @@ Installing DIRAC client
...
-If you want to create a shared client installation, you can do it by simply giving (UNIX) access
-to the directory where the client is installed.
+Python3 client installations
+----------------------------
 
+Choose the directory where you want to install the DIRAC client and::
 
-Install script (for python2 clients)
-------------------------------------
+   $ curl -LO https://github.com/DIRACGrid/DIRACOS2/releases/latest/download/DIRACOS-Linux-$(uname -m).sh
```
Comment (andresailer) [inline docs/source/UserGuide/GettingStarted/InstallingClient/index.rst:17]: Can you add
```
.. highlight:: console
``` somewhere before the first block, this should let one copy the commands without the $ in front, as used also here: https://dirac.readthedocs.io/en/latest/AdministratorGuide/Tutorials/diracSE.html
(reset to python if necessary in this document)

Comment (fstagni) [reply docs/source/UserGuide/GettingStarted/InstallingClient/index.rst:20]: Thanks, done.

File: src/DIRAC/WorkloadManagementSystem/Client/Matcher.py [hunk 2]
```diff
@@ -392,3 +396,22 @@ def _checkPilotVersion(self, resourceDict):
                 "Version check requested but expected project %s != received %s"
                 % (validProject, resourceDict["ReleaseProject"])
             )
+
+
+def parseVersion(releaseVersion):
```
Comment (chaen) [inline src/DIRAC/WorkloadManagementSystem/Client/Matcher.py:401]: Don't we have something very similar in the pilot repo and somewhere else ? 
How about factorizing it in the `diraccfg` repo ?

Comment (fstagni) [reply src/DIRAC/WorkloadManagementSystem/Client/Matcher.py:401]: We do indeed have something very similar for the Pilot, but in the Pilot we can only use "strictly python" as it's code that would run using the local python.

**Comment by fstagni** [comment]
review?

**Review by chaen — APPROVED** [review]

**Pull request merged**